src/fragments/start/getting-started/angular/setup.mdx

```diff
@@ -122,21 +75,6 @@ Angular CLI output warnings: if you see CommonJS or AMD dependencies optimizatio
 
 </Callout>
 
-### Strictly typing `aws-exports`
```

Comment:
I believe this is still necessary since the `aws-exports` file is still a js file.

Reply:
nope, we're switching to JSON!